apps/app-submission-guidelines.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+key: apps-submit-guide
+title: Submission guidelines
+---
+
+We're excited that you want to create an app for ePages!
+Before you start creating it, it's important that you understand our guidelines.
+Our app submission guidelines will help you steer clear of difficulties as you develop your app and provide a structure for you to submit an app that suits our needs.
+The objective of your app should be meaningful and help our customers achieve their goals. Have fun coding!
+
+# Functionality
+
+* Make sure your app performs as described and advertised.
+Apps that crash or have bugs will be rejected.
+* Be transparent with your app documentation and features.
+Apps that include undocumented or hidden features will be rejected.
+* We love to receive the final and production ready version of your app.
+We will not accept demo, trial or test versions.
+* Save your time and effort and don't duplicate an app that is already available in the Apps & Themes Store.
+We may reject it.
+* Concentrate on a reasonable and useful app.
+Apps that provide trick or mock a functionality will be rejected.
+
+# Metadata
+
+* We are keen on complete and meaningful text and descriptions in apps.
+We will not accept apps that contain placeholder text.
```

Comment:
We will not accept apps that contain placeholder text or incomplete information.